### ISSUE TITLE
Handling Signomials ** float

### ISSUE THREAD

**Issue by 1ozturkbe** [hoburg/gpkit#1075]
Currently, gpkit doesn't support constraints such as the one below:
```python
c = (y >= 1./3.*(x-2.)**3. + 0.125*(x-1.)**2. - 2.*x + 10.)
```
This is because it is not clear how you would take the left-hand monomial approximation of the expressions for powers that are not integer. 
Would it make sense to automatically generate the following dummy variables in signomial constraints:
```python
a3 >= x-2.
a2 >= x-1.
```
And sub them into the above constraint to be able to take the approximations of the LHS? 
tl,dr: Are designers always going to have to add dummy variables manually to be able to overcome the Signomial**float problem?

**Comment by 1ozturkbe** [comment]
This is a I-am-curious kind of question; I really don't know what the answer is, or whether or not this is the right approach.

**Comment by bqpd** [comment]
Yeah, good question! A year ago or so, we re-discussed this in the context of Generalized GPs (per Boyd's tutorial) and decided not to implement it because we didn't want to have anything be "magic".

Now that we have ConstraintSets, though, I think this should be implemented as a ConstraintSet!
